src/garage/torch/algos/sac.py

```diff
@@ -377,12 +473,13 @@ def _log_statistics(self, policy_loss, qf1_loss, qf2_loss):
 
         """
         with torch.no_grad():
-            tabular.record('alpha', self.log_alpha.exp().item())
-        tabular.record('policy_loss', policy_loss.item())
-        tabular.record('qf_loss/{}'.format('qf1_loss'), float(qf1_loss))
-        tabular.record('qf_loss/{}'.format('qf2_loss'), float(qf2_loss))
-        tabular.record('buffer_size', self.replay_buffer.n_transitions_stored)
-        tabular.record('local/average_return', np.mean(self.episode_rewards))
+            tabular.record('Train/alpha', self.log_alpha.exp().mean().item())
+        tabular.record('Train/PolicyLoss', policy_loss.item())
```

Comment:
Please use the scheme for logging keys/values used throughout the codebase, e.g. `{AlgoOrModelName}/Quantity/MaybeSubKey`

Reply:
In terms of organization with respect to the reset of the data that is recorded, this organization schema makes more sense. The Train/ stub helps makes the distinction between statistics collected at train time and at evaluation time.

Reply:
You can either rewrite the rest of the codebase to use your scheme, our you can adopt the one I describe here to stay consistent.